packages/toolpad-core/src/SignInPage/SignInPage.tsx

```diff
@@ -242,13 +256,20 @@ function SignInPage(props: SignInPageProps) {
         <Typography variant="body2" color="textSecondary" gutterBottom textAlign="center">
           Welcome, please sign in to continue
         </Typography>
-        <Box sx={{ mt: 2 }}>
+        <Box sx={{ mt: 2, width: '100%' }}>
           <Stack spacing={1}>
-            {error && selectedProviderId !== 'credentials' && selectedProviderId !== 'passkey' ? (
+            {error &&
+            selectedProviderId !== 'credentials' &&
+            selectedProviderId !== 'passkey' &&
+            selectedProviderId !== 'nodemailer' ? (
               <Alert severity="error">{error}</Alert>
             ) : null}
             {Object.values(providers ?? {}).map((provider) => {
-              if (provider.id === 'credentials' || provider.id === 'passkey') {
+              if (
+                provider.id === 'credentials' ||
```

Comment:
You can also just create a single boolean variable for this provider id check, might end up being useful for other situations later too.